src/resources/views/character/contacts.blade.php

```diff
@@ -23,91 +23,54 @@
-@elseif($contact->standing < 0)
-                  text-danger
-@endif
-                  ">
-                <b>{{ $contact->standing }}</b>
-              </span>
-            </td>
-            <td>
-              {{ $labels->whereIn('label_id', $contact->label_ids)->implode('label_name', ', ') }}
-            </td>
-            <td>
-              @if (!in_array($contact->contact_type, ['corporation', 'alliance']))
-                <a href="http://eveskillboard.com/pilot/{{ $contact->contact_id }}"
-                   target="_blank" class="id-to-name" data-id="{{$contact->contact_id}}">
-                  <img src="{{ asset('web/img/eveskillboard.png') }}">
-                </a>
-              @endif
-              @if (!in_array($contact->contact_type, ['corporation', 'alliance']))
-                <a href="http://eve-search.com/search/author/{{ $contact->contact_id }}"
-                   target="_blank" class="id-to-name" data-id="{{$contact->contact_id}}">
-                  <img src="{{ asset('web/img/evesearch.png') }}">
-                </a>
-              @endif
-              @if ($contact->contact_type == 'corporation')
-                <a href="http://evewho.com/corp/{{ $contact->contact_id }}"
-                   target="_blank" class="id-to-name" data-id="{{$contact->contact_id}}">
-                  <img src="{{ asset('web/img/evewho.png') }}">
-                </a>
-              @elseif ($contact->contact_type == 'alliance')
-                <a href="http://evewho.com/alli/{{ $contact->contact_id }}"
-                   target="_blank" class="id-to-name" data-id="{{$contact->contact_id}}">
-                  <img src="{{ asset('web/img/evewho.png') }}">
-                </a>
-              @else
-                <a href="http://evewho.com/pilot/{{ $contact->contact_id }}"
-                   target="_blank" class="id-to-name" data-id="{{$contact->contact_id}}">
-                  <img src="{{ asset('web/img/evewho.png') }}">
-                </a>
-              @endif
-              @if (in_array($contact->contact_type, ['corporation', 'alliance', 'character']))
-                <a href="https://zkillboard.com/{{ $contact->contact_type }}/{{ $contact->contact_id }}"
-                   target="_blank">
-                  <img src="{{ asset('web/img/zkillboard.png') }}">
-                </a>
-              @endif
-              <a href="http://eve-prism.com/?view={{ $contact->contact_type }}&name={{ $contact->contact_id }}"
-                 target="_blank" class="id-to-name" data-id="{{$contact->contact_id}}">
-                <img src="{{ asset('web/img/eve-prism.png') }}"/>
-              </a>
-            </td>
-          </tr>
-        @endforeach
-
-        </tbody>
       </table>
-
     </div>
   </div>
 
 @stop
+
+@push('javascript')
+
+  <script type="text/javascript">
+
+    $('#contact-table').DataTable({
+      "processing": true,
+      "serverSide": true,
+      "ajax": {
+        url: "{{$url}}"
```

Comment:
simply replace the `$url` call by `url()->current()` ?

Reply:
this was very helpful feedback